collect.py

```diff
@@ -202,7 +202,19 @@ def screenshot(driver, file_path):
     image = Image.open(file_path)
     image.save(file_path)
 
-
+    
+def get_code(driver, file_path):
```

Comment:
Split this in two functions, one to store the source of the page and one to store the DOM tree.